### ISSUE TITLE
some problems about reproducing your paper

### ISSUE THREAD

**Issue by lsyl20** [mattpoggi/guided-stereo#6]
For PSM-Net，I set the batchsize to 3 and reduced the learning rate in PSM-Net by four times.10 epochs were then trained on the SceneFlow dataset and subsequently tested on KITTI 2015 with feature enhancement used in training and testing（PSMNet-gd-tr）. The above corresponds to the data in row 8 of Table 2 in the paper.But the results obtained in my experiment are much better than the results in the paper, and my results are as follows:
>2      | >3     | >4     | >5     | EPE
4.865 | 2.515 | 1.712 | 1.316 | 0.758

**Comment by mattpoggi** [comment]
Hi,
probably, your training pipeline is much better than the one we used in the paper. For a fair comparison, we used the training code published by PSMNet authors, which seems not the best choice. We reimplemented the training pipeline from scratch in a different framework and we obtained much better results as well.
Hope this helps

**Comment by lsyl20** [comment]
> 
Thanks for your replay. 
The training pipeline I used is the same as in the PSM-Net source code. The code is modified directly on the source code of PSM-Net, and the hints used for training and testing are all derived from random sampling of radar points in the KITTI2015 training set. The experimental settings are also as mentioned in your paper, batch size=3, other settings are the same as in the PSM-Net algorithm.
